### PR TITLE
[DOCS-6673] Add Alfresco Extension Inspector 1.4

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by Amy-Hyland — APPROVED** [review]
LGTM

**Review by jadeteen989 — APPROVED** [review]

File: content-services/latest/develop/extension-inspector.md [hunk 1]
```diff
@@ -114,12 +117,13 @@ When you run the Inventory command, the output is a report in JSON format with t
...
 The application analyzes custom extensions against WAR inventories.
 
 Use the following command to run the application:
 
 ```bash
-java -jar alfresco-extension-inspector-<version>.jar <extension-filename> [--target-version=6.1.0[-7.0.0] | --target-inventory =/path/to/war_inventory.json] [--verbose=[true | false]]
+java -jar alfresco-extension-inspector-<version>.jar <extension-filename> [--target-version=6.1.0[-7.2.0] | --target-inventory =/path/to/war_inventory.json] [--verbose=[true | false]]
 ```
 
 where:
```
Comment (aborroy) [inline content-services/latest/develop/extension-inspector.md:129]: Adding following text could help to understand what `extension-filename` means:
| extension-filename | Mandatory. AMP or JAR file containing an Alfresco Extension

**Review by aborroy — CHANGES_REQUESTED** [review]
Missing parameter for the analyser option.

**Review by aborroy — APPROVED** [review]
LGTM

**Pull request merged**